library/ssl_tls13_client.c

```diff
@@ -3267,6 +3233,7 @@ static int ssl_server_hello_parse( mbedtls_ssl_context* ssl,
 
 static int ssl_server_hello_postprocess( mbedtls_ssl_context* ssl )
 {
+    int ret;
```

Comment:
```suggestion
    int ret;
    mbedtls_ssl_key_set traffic_keys;
```

Reply:
While no longer strictly necessary since we officially require C99, let's try to stick to having declarations at the beginning of scopes.